### ISSUE TITLE
my_dnn_mitdb.py

### ISSUE THREAD

**Issue by huinee56** [mondejar/ecg_classification_tf#26]
Could you kindly inform me how you train the classification model in the patient-wise mode? I already know that the training set is DS1 and the test set is DS2. Do you use a validation set? If so, how is the validation set obtained? If not, what is your strategy for saving the model?
